webhooks/clients.py

```diff
@@ -4,13 +4,16 @@
 from fastapi_websocket_pubsub import PubSubClient
 import asyncio
 import os
+import sys
 
+sys.path.append(os.path.abspath(os.path.join(os.path.basename(__file__), "..")))
 
 PORT = os.getenv("PORT", "3010")
+URL = os.getenv("WEBHOOKS_URL", "127.0.0.1")
 
 
 async def on_events(data, topic):
-    print(f"{topic}: {data}")
+    print(f"{topic}:\n{data}")
 
 
 async def main():
```

Comment:
What does this method do? Does it subscribe to ACA-Py webhooks? Or is it for others to subscribe to events from the CloudAPI?

Reply:
This clients is only an example of how to roll your own client so people can see how to use it